src/EnergyPlus/OutputProcessor.cc

```diff
@@ -5630,7 +5607,7 @@ void SetupOutputVariable(EnergyPlusData &state,
                                                   thisVarPtr.storeType,
                                                   thisVarPtr.ReportID,
                                                   localIndexGroupKey,
-                                                  TimeStepTypeKey,
+                                                  std::string(sovTimeStepTypeStrings[(int)TimeStepTypeKey]),
```

Comment:
Using a string representation of the time step type enum when passing in the indexGroup.  This is where the HVAC, Zone, System, Plant difference is noted on the output.  If you collapse TimeStepTypeKey into just HVAC and Zone, then this parameter will only be one of those two values, and then the output will be changed...

Reply:
Ah yes, this is where the problem lies. I don't think this should be choosing from TimeStepTypeKey, they are different sets of options.

Reply:
Absolutely, this is where the problems began on that last PR.  If we fix this up independently then going back in and pushing the enum work forward will be super easy.